pom.xml

```diff
@@ -190,12 +211,51 @@
 						<configuration>
 							<excludes>
 								<exclude>**/java11/*Test.java</exclude>
+								<exclude>jdk14/**</exclude>
```

Comment:
We should align the setup for JDK11 and JDK14. Please either move the JDK14 tests to `com.esotericsoftware.kryo.serializers.java14` or the JDK11 tests to a "jdk11" folder.

Reply:
Moved to new jdk11 directory

Reply:
👍